source/guides/deployment.rst

```diff
@@ -34,7 +34,7 @@ The Mattermost server and client apps can run on all of the most popular platfor
 * :doc:`Debian Buster </install/install-debian>`
 * :doc:`RHEL 8 </install/install-rhel-8>`
 * :doc:`CentOS, Oracle Linux, and Scientific Linux </install/install-centos-oracle-scientific>`
-* :ref:`More server install guides <other-resources>`
+* `More server install guides <https://docs.mattermost.com/guides/deployment.html#id1>`_
```

Comment:
@justinegeffen - This is a link to a section located at the bottom of the same page. I can't get this same-page link working. A fresh set of eyes would be most welcome! Note that the preview is currently showing the destination URL as ``https://docs.mattermost.com/guides/deployment.html#id1`` -- Sphinx is likely renaming the URL identifier due to duplicate destination links existing site-wide.

Reply:
Hmmm, I think perhaps a ref:`other resources` could work so I've tried that and will check the preview. I'm not 100% certain my syntax is correct though.